loadsave.c

```diff
@@ -132,15 +133,17 @@ LOAD()
 		RAM[STATUS] = 0;
 		a = 0;
 	} else {
-		FILE *f = fopen(filename, "rb");
+		SDL_RWops *f = SDL_RWFromFile(filename, "rb");
 		if (!f) {
 			a = 4; // FNF
 			RAM[STATUS] = a;
 			status |= 1;
 			return;
 		}
-		uint8_t start_lo = fgetc(f);
-		uint8_t start_hi = fgetc(f);
+		uint8_t start_lo;
+		SDL_RWread(f, &start_lo, sizeof(start_lo), 1);
+		uint8_t start_hi;
+        SDL_RWread(f, &start_hi, sizeof(start_hi), 1);
```

Comment:
I'm curious to know why you don't use `SDL_ReadU8()` in place of `fgetc()`, and `SDL_WriteU8()` in place of `fputc()`.

Reply:
In fact, at this place in the code, you can use
```c
uint16_t start = SDL_ReadLE16(f);
```